Forte.EpiImageHotSpots/Forte.EpiImageHotSpots/ImageHotSpotsEditorDescriptor.cs

```diff
@@ -42,14 +42,14 @@ public override void ModifyMetadata(ExtendedMetadata metadata, IEnumerable<Attri
                 .OrderBy(x => x.Saved)
                 .Last();
 
-            var lastVersion = this.contentLoader.Get<ImageHotSpotsBlockBase>(lastVersionReference.ContentLink);
+            var lastVersion = this.contentLoader.Get<IImageHotSpotsBlock>(lastVersionReference.ContentLink);
```

Comment:
since there's no [AllowedTypes] restriction anymore and it's up to end user if he puts it or not I believe we need TryGet here as lastVersionReference might be anything in this case?

Reply:
Not exactly. We have a Block which has a ContentArea. Here we are trying to get the Block itself. I am chaching if it was published at all in line 38.
In line 54 I am referencing to the items from the ContentArea using 
`this.contentLoader.Get<IContent>(x.ContentLink)`
so I think we're pretty safe